app/controllers/v0/appeals_controller.rb

```diff
@@ -12,5 +12,21 @@ def index
         json: appeals_response.body
       )
     end
+
+    def show
+      service.get_higher_level_reviews params[:uuid]
+      render json: higher_level_review
+    end
+
+    def intake_status
+      intake_status = service.get_higher_level_reviews_intake_status params[:uuid]
+      render json: intake_status
+    end
+
+    private
+
+    def service
+      DecisionReview::Service.new
```

Comment:
Might we define the `DecisionReview` service namespaced under `Appeals` so as to share configuration? It connects to the same backend service (Caseflow) right?

Reply:
I think this suggestion makes sense. Wouldn't be a bad idea to raise this with Silvio in his PR, where the service integration work is defined. [PR 3439](https://github.com/department-of-veterans-affairs/vets-api/pull/3439)